doc/releases/changelog-dev.md

```diff
@@ -26,4 +29,5 @@
 This release contains contributions from (in alphabetical order):
 
 Abhishek Abhishek,
-Matthew Silverman.
\ No newline at end of file
+Matthew Silverman,
+Sergei Mironov.
```

Comment:
```suggestion
Sergei Mironov,
Matthew Silverman,
```

Reply:
Not important for me, but does the comment says it should be in alphabetical order?

Reply:
Not relevant anymore: release branch has a different changelog file.